pom.xml

```diff
@@ -94,7 +87,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- sonarqube properties -->
-        <sonar.organization>fortitudetec</sonar.organization>
+        <sonar.organization>elucidation-project</sonar.organization>
```

Comment:
Don't we also need the `sonar.projectKey`? Though, I tried doing that in a separate PR and it didn't like it. Complained that you cannot have two of them...